packages/mobile/src/web3/dataEncryptionKey.ts

```diff
@@ -103,10 +126,53 @@ export function* registerAccountDek(account: string) {
     }
 
     // Generate and send a transaction to set the DEK on-chain.
-    const setAccountTx = accountsWrapper.setAccount('', publicDataKey, account)
+    let setAccountTx = accountsWrapper.setAccount('', publicDataKey, walletAddress)
```

Comment:
If you just need to register the DEK, suggest calling `setAccountDataEncryptionKey`. If you just need to register the wallet address, suggest calling `setWalletAddress`. Only if you need to set both is it best to use `setAccount`

Reply:
Yeah we need to set both. I will update the naming to reflect that in a subsequent PR.